misc/parity/json_diff.py

```diff
@@ -171,3 +177,29 @@ def pd(jd):
         return jd
 
     return sort_json(pd(json_diff))
+
+
+def select(d: dict, where: Union[dict, list]):
+    """
+    Keep only paths in d modeled by where.
+    If where is empty, keep everything.
+    """
+    if not where:
+        return d
+
+    if not isinstance(d, dict):
```

Comment:
I'm trying to be super relaxed here. So writing it this way allows you to have something like:
```python
where = {
   "definitions": ["Accounts"],
   "endpoints": False,
}
```

which would be treated the same as:
```python
where = {
   "definitions": ["Accounts"],
   "endpoints": {},
}
```

or the same as

```python
where = {
   "definitions": ["Accounts"],
   "endpoints": [],
}
```

Since this is all just a big hack, I'm not super concerned, but if others start messing with it, maybe it's worth clarifying.